src/web/components/SubNav/SubNav.tsx

```diff
@@ -1,87 +1,207 @@
+    subNavSections?: SubNavType;
     pillar: Pillar;
     currentNavLink: string;
-}
+};
+
+const wrapperCollapsed = css`
+    height: 36px;
+    overflow: hidden;
 
-export class SubNav extends Component<
-    Props,
-    {
-        showMore: boolean;
-        isExpanded: boolean;
+    ${from.tablet} {
+        height: 42px;
     }
-> {
-    private ulRef: React.RefObject<HTMLUListElement>;
-
-    constructor(props: Props) {
-        super(props);
-        this.state = { showMore: false, isExpanded: false };
-        // More should not be shown without JS
-        this.ulRef = createRef();
+`;
+
+const subnav = css`
+    list-style: none;
+    padding: 0 5px;
+
+    ${from.mobileLandscape} {
+        padding: 0 15px;
     }
 
-    public componentDidMount() {
-        // If componentDidMount runs we know client-side JS is enabled
-        this.setState({
-            showMore: this.shouldShowMore(),
-        });
+    li {
+        float: left;
+        display: block;
     }
+`;
 
-    public toggle = () => {
-        this.setState(prevState => ({
-            isExpanded: !prevState.isExpanded,
-        }));
-    };
-
-    public shouldShowMore() {
-        // get ul ref
-        const ul = this.ulRef.current;
-        if (!ul) {
-            return false;
-        }
+const subnavExpanded = css`
+    ${subnav};
+`;
+
+const subnavCollapsed = css`
+    ${subnav};
+    max-width: calc(100% - 60px);
 
-        const lis = ul.querySelectorAll('li');
-        const lastLi = lis[lis.length - 1];
+    ${from.mobileLandscape} {
+        max-width: calc(100% - 70px);
+    }
+`;
 
-        const ulTop = ul.getBoundingClientRect().top;
-        const liTop = lastLi.getBoundingClientRect().top;
+const fontStyle = css`
+    ${textSans.medium()};
+    font-weight: 500;
+    color: ${palette.neutral[7]};
+    padding: 0 5px;
+    height: 36px;
+    /* Design System: Line height is being used here for centering layout, we need the primitives */
+    line-height: 36px;
 
-        return ulTop !== liTop;
+    ${from.tablet} {
+        height: 42px;
+        /* Design System: Line height is being used here for centering layout, we need the primitives */
+        /* stylelint-disable-next-line property-blacklist */
+        line-height: 42px;
     }
+`;
 
-    public render() {
-        const { subnav, pillar, currentNavLink } = this.props;
+const linkStyle = css`
+    ${fontStyle};
+    float: left;
+    text-decoration: none;
 
-        if (!subnav) {
-            return null;
+    :hover {
+        text-decoration: underline;
+    }
+`;
+const selected = css`
+    font-weight: 700;
+`;
+const moreStyle = css`
+    ${fontStyle};
+
+    cursor: pointer;
+    border: none;
+    background-color: transparent;
+    color: ${text.supporting};
+
+    :hover {
+        color: ${palette.news.main};
+    }
+
+    ${from.desktop} {
+        display: none;
+    }
+`;
+
+const parentLinkStyle = css`
+    ${linkStyle};
+    font-weight: 700;
+`;
+const ps1 = css`
```

Comment:
Coming at this cold, I'm at a loss what ps1 means. I know these are existing names but it'd be great to include some more meaningful ones as part of the refactor

Reply:
Yeah idk what this is meant to be referring to, I will try find a better name